cron/expire.js

```diff
@@ -3,6 +3,10 @@
 /**
  * Test expiration date of user, if expired, expire the user
  */
+
+const { promisify } = require('util');
```

Comment:
where does this come from ?

util/promisify?

Reply:
https://nodejs.org/api/util.html#utilpromisifyoriginal

Reply:
in fact it allow to do a await on setimeout to simulate sleep
I have tested it and it work